raft/src/main/java/org/apache/kafka/raft/MetadataLogConfig.java

```diff
@@ -52,14 +51,14 @@ public class MetadataLogConfig {
             "configuration. The Kafka node will generate a snapshot when either the maximum time interval is reached or the " +
             "maximum bytes limit is reached.";
 
-    public static final String METADATA_LOG_SEGMENT_MIN_BYTES_CONFIG = "metadata.log.segment.min.bytes";
-    public static final String METADATA_LOG_SEGMENT_MIN_BYTES_DOC = "Override the minimum size for a single metadata log file. This should be used for testing only.";
-    public static final int METADATA_LOG_SEGMENT_MIN_BYTES_DEFAULT = 8 * 1024 * 1024;
-
     public static final String METADATA_LOG_SEGMENT_BYTES_CONFIG = "metadata.log.segment.bytes";
     public static final String METADATA_LOG_SEGMENT_BYTES_DOC = "The maximum size of a single metadata log file.";
     public static final int METADATA_LOG_SEGMENT_BYTES_DEFAULT = 1024 * 1024 * 1024;
 
+    public static final String INTERNAL_METADATA_LOG_SEGMENT_BYTES_CONFIG = "internal.metadata.log.segment.bytes";
```

Comment:
Do we need another internal config? User can configure `internal.segment.bytes` if they want to create small segment, right?